erpnext/stock/doctype/stock_reconciliation/stock_reconciliation.py

```diff
@@ -31,6 +31,7 @@ def validate(self):
 		self.validate_expense_account()
 		self.set_total_qty_and_amount()
 		self.validate_putaway_capacity()
+		self.validate_customer_provided_item()
 
```

Comment:
Can you also additionally add a function to set a customer provided Item's valuation rate to zero?

In other transactions even if the user accidentally adds a rate, the system automatically sets it to zero again.
Lets do that here to make sure it never accidentally has a valuation.